azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/MultiOracleScenarioRunner.kt

```diff
@@ -55,8 +55,7 @@ class MultiOracleScenarioRunner<
         QF : QueryFactory,
         AGF : ActionGeneratorFactory,
         >(
-        val answers: List<Answer<*, CF>>?,
-        val scenario: OracleScenario<AF, QF, AGF>
+        val answers: List<Answer<*, CF>>?, val scenario: OracleScenario<AF, QF, AGF>
```

Comment:
I feel sure that the formatter has become more opinionated in several places in this file :smile: